examples/src/main/java/com/mapbox/navigation/examples/core/TripServiceActivityKt.kt

```diff
@@ -36,6 +36,12 @@ import kotlinx.coroutines.isActive
 import kotlinx.coroutines.launch
 import timber.log.Timber
 
+/**
+ * This example shows how to use the Navigation SDK's
+ * [MapboxTripService] to push update the Android-system
```

Comment:
To push update the Android-system notification? 🤔

Reply:
Fixed